service/src/test/java/bio/terra/workspace/pact/LzApiTest.java

```diff
@@ -0,0 +1,612 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.http.HttpStatus;
+
+@Tag("pact-test")
+@ExtendWith(PactConsumerTestExt.class)
+public class LzApiTest {
+
+  HttpLandingZoneService landingZoneService;
+
+  static final Map<String, String> contentTypeJsonHeader =
+      Map.of("Content-Type", "application/json");
+  static final UUID landingZoneId = UUID.fromString("a051c5f8-9de8-4b4f-a453-8f9645df9c7b");
+  static final UUID asyncJobId = UUID.fromString("3b0db775-1916-42d2-aaeb-ebc7aea69eca");
+  static final UUID billingProfileId = UUID.fromString("4955757a-b027-4b6c-b77b-4cba899864cd");
+  static final UUID azureResourceId = UUID.fromString("e3327645-1e8f-454d-bdae-fe06b4762542");
+
+  static DslPart jobReportShape =
+      new PactDslJsonBody()
+          .stringType("id")
+          .stringType("description")
+          .stringMatcher("status", "RUNNING|SUCCEEDED|FAILED")
+          .integerType("statusCode")
+          .datetime("submitted")
+          .datetime("completed")
+          .stringType("resultURL");
+
+  static DslPart errorReportShape =
+      new PactDslJsonBody().stringType("message").integerType("statusCode").object("causes");
+
+  static DslPart jobControlShape = new PactDslJsonBody().stringType("id");
+
+  static DslPart landingZoneCreateRequestShape =
+      new PactDslJsonBody()
+          .uuid("landingZoneId")
+          .stringType("definition")
+          .stringType("version")
+          .uuid("billingProfileId")
+          .object("jobControl", jobControlShape)
+          .eachLike("parameters")
+          .stringType("key")
+          .stringType("value")
+          .closeArray();
+
+  static final DslPart resourceShape =
+      new PactDslJsonBody()
+          .stringType("resourceId")
+          .stringType("resourceType")
+          .stringType("resourceName")
+          .stringType("resourceParentId")
+          .stringType("region")
+          .object("tags");
+
+  static final DslPart resourcesByTypeShape =
+      new PactDslJsonBody()
+          .stringType("description")
+          .stringType("purpose", "SHARED_RESOURCE")
+          .eachLike("deployedResources", resourceShape);
+
+  @BeforeEach
+  void setup(MockServer mockServer) {
+    var config = mock(LandingZoneServiceConfiguration.class);
+    when(config.getBasePath()).thenReturn(mockServer.getUrl());
+    landingZoneService = new HttpLandingZoneService(OpenTelemetry.noop(), config);
+  }
+
+  @Pact(consumer = "workspacemanager", provider = "lzs")
+  public RequestResponsePact startCreateLandingZone(PactDslWithProvider builder) {
+    var createResponseShape =
+        new PactDslJsonBody()
+            .uuid("landingZoneId")
+            .stringType("definition")
+            .stringType("version")
+            .object("jobReport", jobReportShape)
+            .object("errorReport", errorReportShape);
```

Comment:
Same thing as above, about optional fields.
I wouldn't expect the `errorReport` to be present in the initial create response - so it shouldn't be here.